cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,74 @@
 
 describe('Book Store app', () => {
   before(() => {
+    cy.visit('/login');
+  });
+  // const fName = 'testName';
+  // const lName = 'testSurname';
+
+  const uName = 'testUsername';
+  const uPassword = 'testPassword1!';
+  it('correct login user', () => {
+    cy.get('#userName')
```

Comment:
You can use your `login()` function here too.
If you do so, you don't need variables declared in lines 10. and 11.

Reply:
fixed